package.json

```diff
@@ -97,6 +96,10 @@
     "universal-cookie": "^4.0.3",
     "use-deep-compare-effect": "^1.3.1"
   },
+  "peerDependencies": {
+    "react": "^16.14.0",
+    "react-dom": "^16.14.0"
+  },
```

Comment:
We don't actually want to supply copies of React and ReactDOM to dependents (like portals). They should be peerDependencies.